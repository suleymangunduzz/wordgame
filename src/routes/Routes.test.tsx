import React from 'react';
import { shallow } from 'enzyme';

import Routes from './Routes';

const wrapper = shallow(<Routes />);

it('renders correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
