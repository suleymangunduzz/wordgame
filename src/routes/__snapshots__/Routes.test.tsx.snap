// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Switch>
`;
